src/components/NFTPage/CollectorList/index.vue

```diff
@@ -69,17 +68,22 @@ export default {
       type: Number,
       default: undefined,
     },
-    ownerList: {
+    items: {
       type: Array,
       default: undefined,
     },
-    populatedList: {
-      type: Array,
-      default: undefined,
+  },
+  computed: {
+    trimmedItems() {
+      if (this.items.length >= 10) {
+        return this.items.filter((id, index) => index <= 9).map(owner => ({
+          ...owner,
+        }));
+      }
+      return undefined;
     },
```

Comment:
Suggest to add a prop `trimmedCount` with default value `10`

return the list with `Array.slice()` e.g.`items.slice(0, trimmedCount)`
no need to return `undefined`